sourcegraph/sourcegraph.proto

```diff
@@ -3005,3 +3005,43 @@ service GraphUplink {
 	// instance
 	rpc PushEvents(UserEventList) returns (pbtypes.Void);
 }
+
+// NotifyGenericEvent describes an action being done against an object. For
+// example reviewing a changeset.
+message NotifyGenericEvent {
+	// Actor is the User who did the action
+	UserSpec actor = 1;
+
+	// Recipients is who should be notified of the action
+	repeated UserSpec recipients = 2;
+
+	string action_type = 3;
+	string action_content = 4;
+	string object_repo = 5;
+	string object_name = 6;
+	string object_url = 7 [(gogoproto.customname) = "ObjectURL"];
```

Comment:
This is consistent with the rest of this file (look at other similar fields that have an acronym in the name). It's needed because by default it would end up being `ObjectUrl` rather than `ObjectURL` as mandated by Go style.